products/compute/terraform.yaml

```diff
@@ -504,8 +504,15 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       - !ruby/object:Provider::Terraform::Examples
         name: "disk_basic"
         primary_resource_id: "default"
+        primary_resource_name: "fmt.Sprintf(\"tf-test-test-disk%s\", context[\"random_suffix\"])"
```

Comment:
This is interesting - we don't usually let ourselves write go code in these yamls.  Is this normal / added since the last time I looked at these?

Reply:
I am not sure if this is normal or not, but it seems widely used. In the same file, you may see similar usages in line 936, 986 and 2368. They appear to be all for IAM examples related